bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadEntryProcessor.java

```diff
@@ -105,9 +105,7 @@ protected void processPacket() {
             }
             errorCode = BookieProtocol.ENOENTRY;
         } catch (IOException e) {
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Error reading {}", request, e);
-            }
+            LOG.error("IOException while reading {}", request, e);
```

Comment:
I believe this would print the whole stack trace every time a client gets a connectivity error with any bookie, for each entry that we're trying to read.

Reply:
> whole stack trace every time a client gets a connectivity error with any bookie

Thanks for your review @merlimat . But I don't get your point. This method runs only on serever side and probability throw `IOException` if read an entry from storage. So I think it doesn't matter with client print logs

Reply:
Pring `ledgerId:entryId` instead of the whole request content?

Reply:
I think print  whole `request ` is OK.  because `ReadRequest` has overwrite the `toString` method:
https://github.com/apache/bookkeeper/blob/34bd4b38fa13f105608ae4e7b4c749124c271e1e/bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java#L249-L251
@hangc0276

Reply:
>I believe this would print the whole stack trace every time a client gets a connectivity error with any bookie, for each entry that we're trying to read.

If the target entry does not exist on the bookie, it will throw NoLedgerException or NoEntryException and won't go into this code path. @merlimat Do you have any concern?

Reply:
> > I believe this would print the whole stack trace every time a client gets a connectivity error with any bookie, for each entry that we're trying to read.
> 
> If the target entry does not exist on the bookie, it will throw NoLedgerException or NoEntryException and won't go into this code path. @merlimat Do you have any concern?

The NoLedgerException and NoEntryException extend IOException, it should be caught.